go.mod

```diff
@@ -33,6 +33,7 @@ require (
 	github.com/certifi/gocertifi v0.0.0-20210507211836-431795d63e8d // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/etdub/goparsetime v0.0.0-20160315173935-ea17b0ac3318 // indirect
+	github.com/google/shlex v0.0.0-20191202100458-e7afc7fbc510 // indirect
```

Comment:
it seems to be an archived project. I would not mind either copying and pasting it directly in as it is pretty small, or finding a different project with better support.